src/components/marquee-wizard/marquee-wizard-step.component.ts

```diff
@@ -51,4 +36,10 @@ export class MarqueeWizardStepComponent extends WizardStepComponent {
         this.completed = completed;
         this.completedChange.emit(completed);
     }
+
+    ngOnChanges(changes: SimpleChanges): void {
```

Comment:
Could this cause any issues if someone was to programmatically change the valid value rather than using the Input? As this will not get called?

Reply:
Good spot, this won't work. I'll find a more suitable solution